frontend/src/views/CellCards/components/CellCard/components/OntologyDagView/common/constants.ts

```diff
@@ -0,0 +1,13 @@
+export const primaryColor = "#0073FF";
```

Comment:
color, size, margin, and animation constants used in the ontology tree visualization